test/cypress/components/multiselect.js

```diff
@@ -1,15 +1,18 @@
 export class Multiselect {
 
-  constructor() {
+  constructor( label ) {
```

Comment:
Seems like the test components don't have commentary, but wouldn't hurt to add a jsdoc above this saying what the label is passed in for.